src/test/scala/io/iohk/ethereum/consensus/validators/std/StdOmmersValidatorSpec.scala

```diff
@@ -77,7 +92,29 @@ class OmmersValidatorSpec extends AnyFlatSpec with Matchers with ScalaCheckPrope
   // scalastyle:off magic.number
   trait BlockUtils extends EphemBlockchainTestSetup {
 
-    val ommersValidator = new StdOmmersValidator(blockchainConfig, new EthashBlockHeaderValidator(blockchainConfig))
+    val ommersValidator = new StdOmmersValidator(new EthashBlockHeaderValidator(blockchainConfig))
+
+    val ommerInvalidBranch = BlockHeader(
+      parentHash = ByteString(Hex.decode("fd07e36cfaf327801e5696134b12345f6a89fb1e8f017f2411a29d0ae810ab8b")),
```

Comment:
Why did all this have to be added? What do all the scary hex strings mean?

Reply:
It's for test I added. The scary hex'es are hashes :)

Reply:
Could these hashes be computed on the fly so you can see what's inside? Are these from existing tests?

Reply:
These are not random hashes, they validate the block. If you notice I had to write a different test, than the rest of the tests in the spec, trying to just make one hash invalid was breaking the whole block and the validation was failing before validating the ommers.
I did copy this data from another test, yes, and it made me realize that getting this kind of data is not trivial. Maybe it's available in block explorers, I did not check, it just crossed my mind